lib/task_helpers/exports.rb

```diff
@@ -0,0 +1,41 @@
+module TaskHelpers
+  class Exports
+    # Illegal characters: '/', '|'
+    REGEX_KEEP_SPACES   = %r{[/|]}
+
+    # Illegal characters: '/', '|', ' '
+    REGEX_REMOVE_SPACES = %r{[/| ]}
+
+    # Replacement character
+    REPLACEMENT         = '_'.freeze
+
+    # Replace characters in \p str that are not allowed in filenames.
+    # Illegal characters are matched from the regexes REGEX_REMOVE_SPACES
+    # (if \p keep_spaces = false ) REGEX_KEEP_SPACES (if \p keep_spaces = true).
+    # Illegal characters are replaced with @@replacement.
+    # @param [String] str string to replace
+    # @param [Hash] options If options[:keep_spaces] == true, then keep spaces in
+    # \p str , otherwise replace spaces. (default = false, replace spaces)
+    def self.replace_chars(str, options = {})
+      if options.key?(:keep_spaces) && options[:keep_spaces] == true
+        str.gsub(REGEX_KEEP_SPACES, REPLACEMENT)
+      else
+        str.gsub(REGEX_REMOVE_SPACES, REPLACEMENT)
+      end
+    end
+
+    def self.parse_options
+      require 'trollop'
+      options = Trollop.options(EvmRakeHelper.extract_command_options) do
+        opt :keep_spaces, 'Keep spaces in filenames', :type => :boolean, :short => 's', :default => false
+        opt :directory, 'Directory to place exported files in', :type => :string, :required => true
+      end
+
+      # Check for a valid directory
+      Trollop.die :directory, 'Destination directory must already exist' unless File.directory?(options[:directory])
+      Trollop.die :directory, 'Destination directory must be writeable' unless File.writable?(options[:directory])
+
+      options.delete_if { |_k, v| v.nil? || v == false }
```

Comment:
I don't think it is necessary either. It came as part of the copying code from https://github.com/ManageIQ/manageiq/blob/22efb0914107963b95e064c9d7adb2aa6d905e2a/lib/tasks/evm_dba.rake for Trollop.